apertium-languages.tsv

```diff
@@ -24,6 +24,7 @@ cos,42,9
 crh,45.3,34.4
 cym,52.5,-4
 dan,55.5,11.5
+deu,52,10
```

Comment:
Where did you get this coordinate from?

Reply:
Hmm, can you change to 51,9? You should just get the coordinate of the entire country through Wikipedia, not some city in the country.

Reply:
Sure, I did that

Reply:
Should I add dut to it now?

Reply:
@dolphingarlic Since dut is the same as nld, we will just have two nodes that are on top of each other which isn't good. Anyways, I think your code is already converting the dut pairs to nld pairs which is fine.

Reply:
I take that back, your code isn't converting dut pairs to nld pairs. There are no dut pairs at all in the pairs.txt.json file so there is no need to add it since nld already does the job.